src/components/Operations/SortRemoveMotion/sort-remove--motion.module.scss

```diff
@@ -0,0 +1,135 @@
+
+
+@keyframes sort-remove-line1 {
+  0% {
+    transform: translateX(0px);
+  }
+  10% {
+    transform: translateX(6px);
+  }
+  20% {
+    transform: translateX(0px);
+  }
+  100% {
+    transform: translateX(0px);
+  }
+}
+
+@keyframes sort-remove-line2 {
+  0% {
+    transform: translateX(0px);
+  }
+  10% {
+    transform: translateX(-6px);
+  }
+  20% {
+    transform: translateX(0px);
+  }
+  100% {
+    transform: translateX(0px);
+  }
+}
+
+@keyframes sort-remove-line3 {
+  0% {
+    transform: translateX(0px);
+  }
+  10% {
+    transform: translateX(-2px);
+  }
+  20% {
+    transform: translateX(0px);
+  }
+  100% {
+    transform: translateX(0px);
+  }
+}
+
+
+@keyframes sort-remove-vertical {
+  0% {
+    transform: scaleY(1);
+  }
+  10% {
+    transform:  scaleY(.7);
+  }
+  20% {
+    transform: scaleY(1);
+  }
+  100% {
+    transform: scaleY(1);
+  }
+}
+
+@keyframes sort-remove-horizontal {
+  0% {
+    transform: scaleX(1);
+  }
+  10% {
+    transform:  scaleX(.7);
+  }
+  20% {
+    transform: scaleX(1);
+  }
+  100% {
+    transform: scaleX(1);
+  }
+}
```

Comment:
Not sure if you’ve run into this before with other icons, but noticed that at 100% (default zoom) this appears rotated (like a <kbd>+</kbd> sign) in Chrome. This was not an issue in Safari though so it’s possible that it’s specific to Chrome? As is though, this could be a confusing animation.

![2022-12-12 at 16 45 09 - sort remove](https://user-images.githubusercontent.com/1717398/207161589-d0d68165-7743-4515-ab06-19ab62d0ca78.gif)

(Zoomed in at the end where it does not appear to be an issue anymore.)

Reply:
@elycheea Thanks for catching this and for all the help! I can't seem to recreate the glitch in Chrome at 100 zoom or zoomed out/in.  Hmm. Weird! I'll keep trying.

@kristastarr Do this glitch happen for you too in chrome?

Reply:
Fwiw, I’m in Chrome 108. Probably won’t get a chance to look into it more today, but I definitely did not see this in any of the other close/X icons when I did a quick check.

Reply:
Very weird. I don't see the problem zoomed in, out, or at default zoom in Chrome 108, Firefox, or Safari 🤔  I've been doing some reading on `transform: rotate()` vs `rotate`.  I'm going to try changing to `transform: rotate` to see if we get a different result - @elycheea after I push the change, would you mind checking Chrome again and see if you are still seeing the issue?

Reply:
Well I'm super confused now bc when I make the change from `rotate: 45deg` to `transform: rotate(45deg)`, I am seeing the issue.  I went ahead and pushed the change.  @johnbister @elycheea do you both want to check and see how it looks for you now?

Reply:
@kristastarr @elycheea I also see the issue now that it has been changed to `transform: rotate(45deg)`

I still don't can't recreate the issue at any zoom or default in Chrome when it is `rotate: 45deg`. So weird. Should we change it back?